legacy/app/main/map/map-module.js

```diff
@@ -0,0 +1,47 @@
+angular.module('ushahidi.map', [])
+
+// Timeline, data and Map screen
+.directive('postViewMap', require('./post-view-map.directive.js'))
+
+//Mode context
+.directive('modeContext', require('./mode-context/mode-context.directive.js'))
+.directive('modeContextFormFilter', require('./mode-context/mode-context-form-filter.directive.js'))
+.directive('filterBySurveyDropdown', require('./mode-context/filter-by-survey-dropdown.directive.js'))
+.directive('filterByDatasource', require('./mode-context/filter-by-datasource.directive.js'))
+
+.directive('addPostButton', require('./post-toolbar/add-post-button.directive.js'))
+.directive('addPostSurveyList', require('./post-toolbar/add-post-survey-list.directive.js'))
+.directive('addPostTextButton', require('./post-toolbar/add-post-text-button.directive.js'))
+
+// Post editing workflows
+.directive('filterPostSortingOptions', require('./post-toolbar/filters/filter-post-sorting-options.directive.js'))
+.directive('filterPostOrderAscDesc', require('./post-toolbar/filters/filter-post-order-asc-desc.directive.js'))
+.directive('filterUnlockedOnTop', require('./post-toolbar/filters/filter-unlocked-on-top.directive.js'))
+
+// Filters
+.directive('filterPosts', require('./post-toolbar/filters/filter-posts.directive.js'))
```

Comment:
@Ifycode add `.directive('postToolbar', require('./post-toolbar/post-toolbar.directive.js'))` below line 21 and it should work again 👍

Reply:
Thank you @Angamanga